docs/source/getting_started/advanced.rst

```diff
@@ -229,6 +229,30 @@ For more information check the documentation:
 
 
 
+Obtaining Good Embeddings
+---------------------------
+
+We optimized the workflow of sampling only important datapoints by using **low-dimensional embeddings**. 
+This has two benefits:
+
+- Low-dimensional embeddings have more meaningful distance metrics. 
+  We know that the data usually lies on a manifold in high-dimensional spaces 
+  (see `curse of dimensionality <https://en.wikipedia.org/wiki/Curse_of_dimensionality>`_). 
+  Even very similar samples might have a high L2-distance or low cosine similarity in high embeddings.
+- Most algorithms to sample a subset based on the embeddings scale with 
+  the dimensionality. Therefore low-dimensional embeddings can significantly 
+  reduce computing time.
+
+We leverage **self-supervised learning** to obtain good 
```

Comment:
do we have a section about SSL? would be great to link it there or to wikipedia. Wdyt?

Reply:
We will have a section in the future - would add the link then